zeno/mirror/sdm.py

```diff
@@ -0,0 +1,24 @@
+import json
+import os
+
+import pandas as pd
+
+
+EXAMPLE_SLICES = False
+
+
+def domino(cache_path, pd_with_preds_embed: pd.DataFrame, num_slices: int = 10):
```

Comment:
So is this hardcoded currently?

Reply:
(totally fine that it is, just curious)

Reply:
Yeah I'm running it in a colab notebook to experiment right now. And it actually has a gpu so it doesn't take 10 years on my laptop.